.github/workflows/publish-docs.yml

```diff
@@ -1,57 +1,44 @@
 on:
   workflow_dispatch:
     inputs:
-      workspace_path:
-        description: 'Path to the workspace being released.'
+      pkg_name:
+        description: 'The package to publish'
         required: true
-        type: string
-      project_file:
-        description: 'Path to the project file.'
-        required: true
-        type: string
-      test_project_file:
-        description: 'Path to the test project file.  If provided, run unit tests, otherwise skip them.'
-        required: false
-        type: string
-  workflow_call:
-    inputs:
-      workspace_path:
-        description: 'Path to the workspace being released.'
-        required: true
-        type: string
-      project_file:
-        description: 'Path to the project file.'
-        required: true
-        type: string
-      test_project_file:
-        description: 'Path to the test project file.  If provided, run unit tests, otherwise skip them.'
-        required: false
-        type: string
+        type: choice
+        options:
+          - LaunchDarkly.ClientSdk
+          - LaunchDarkly.ServerSdk
+          - LaunchDarkly.ServerSdk.Telemetry
```

Comment:
For reviewers: This gives a dropdown when triggering the action, a better user experience with less chance for errors.